content/lessen/wiskunde/rekenen_rationaal/rekenen_gehele_getallen/index.md

```diff
@@ -0,0 +1,122 @@
+
+Merk op dat we $-2$ tussen haakjes hebben geschreven. Dit doen we om verwarring te voorkomen. We tellen het getal $-2$ op bij $10$.
+
+### Wat te doen met verschillende tekens?
+Soms kom je uitdrukking tegens zoals $2 -(-4))$. Wat moet je doen met al deze tekens en hoe bepaal je de uitkomst? Daarvoor kan je je best een persoon
+langs een getallenas voorstellen. 
+
+Een **cijfer** vertelt ons het **aantal stappen** dat deze persoon zet. De richting {{< mute "(links of rechts)" >}} wordt bepaald
+door het teken. We starten standaard naar rechts, zoals we ook van links naar rechts schrijven. Als we een **plusteken** tegenkomen in de uitdrukking, dan gaan we
+verder **in dezelfde richting**. Komen we echter een **minteken** tegen, dan draaien we om een gaan we **in de andere richting** verder.
+
+{{< svg "img/verschillende_tekens.svg" "Een persoon die langs de getallenas beweegt." >}}
+
+De persoon die de bewerking hierboven uitvoert langs de getallenas begint dus met **twee stappen naar rechts** te zetten. Vervolgens draait hij zich om, waarna hij zich
+nogmaals omdraait en dan **vier stappen (naar rechts)** zet.
+
+## Gehele getallen vermenigvuldigen
+
+{{< svg "img/vermenigvuldiging.svg" "Een vermenigvuldiging bestaat uit factoren die als resultaat een product geven. We gebruiken het maalteken." >}}
+
+Een vermenigvuldiging is eigenlijk een manier om een **optelling van dezelfde getallen korter** te schrijven.
+
+$$10 \cdot \orange{2} = \underbrace{10 + 10}_{\orange{2}\ \text{keer}} = 20$$
+
+Welk teken denk je dat $3 \cdot (-2)$ zal hebben? Als je dit uitschrijft zoals hierboven, kan je dit makkelijk te weten komen!
+
+$$\orange{3} \cdot (-2) = \underbrace{(-2) + (-2) + (-2)}_{\orange{3}\ \text{keer}} = -6$$
+
+Ook een vermenigvuldiging met **twee negatieve factoren** kan je op deze manier schrijven. Je moet de getallen dan niet optellen,
+maar **aftrekken**.
+
+$$\orange{-3} \cdot (-4) = \underbrace{\green{-}(-4) - (-4) - (-4)}_{\orange{-3}\ \text{keer}} = 12$$
+
+Nu vraag je je misschien af vanwaar de $\green{-}$ aan het begin komt. Eigenlijk kunnen we steeds een teken schrijven voor de eerst term,
+maar bij de eerste twee voorbeelden maakt dit geen verschil. 
+
+$$10 \cdot \orange{2} = \underbrace{\green{+}10 + 10}_{\orange{2}\ \text{keer}} = 20$$
+
+$$\orange{3} \cdot (-2) = \underbrace{\green{+}(-2) + (-2) + (-2)}_{\orange{3}\ \text{keer}} = -6$$
+
+### Negatieve getallen
+
+Hieruit kunnen we besluiten dat als er een **even aantal negatieve factoren** zijn,
+de uitkomst **positief** is. Als er een **oneven aantal negatieve factoren** zijn, dan is de uitkomst **negatief**.
+
+> Nog niet overtuigd :thinking_face:? Bekijk de alinea *Wat te doen met verschillende tekens?* nog eens grondig!
+
+## Gehele getallen delen
+
+{{< svg "img/deling.svg" "Een deling bestaat uit een deeltal en een deler die als resultaat een quotiënt geven. We gebruiken het deelteken." >}}
+
+Een deling is eigenlijk het omgekeerde van een vermenigvuldiging.
```

Comment:
Hier de getallenas ook weer bij betrekken. We zouden kunnen zeggen dat we een vermenigvuldiging ook kunnen zien als grotere stappen zetten. $6 \cdot 3$ is zes stappen zetten met stappen die drie keer groter zijn dan normaal. Een deling is het omgekeerde waarbij we bij $6 : 3$ zes stappen zetten met stappen die drie keer kleiner zijn dan normaal.

Reply:
Nog niet 100% zeker of deze analogie wel zo nuttig is.

Reply:
Ja ik had ook al aan zoiets gedacht, met een zeker basis (uit de lineaire algebra), maar dit kan misschien voor velen verwarrender zijn dan dat het gaat helpen.